packages/app/src/components/signed-out/login-form.tsx

```diff
@@ -30,10 +31,15 @@ function LoginForm({ className, ...props }: Readonly<React.ComponentPropsWithout
 
   const onSubmit = async (credentials: LoginFormValues) => {
     try {
-      await signIn(credentials);
```

Comment:
We were still redirecting, even on failed login attempt. This update will prevent that from happening.